### PR TITLE
Update to inaccurate footer text

### PR DESCRIPTION
**Pull request by CasparN**

### CODE REVIEW

**Comment by Bluesy1** [comment]
Have you checked if this is correct for tictactoe?

**Comment by CasparN** [comment]
TicTacToe doesn't have the ``Type /tictactoe to play`` or something along those lines. It doesn't even have a footer.

**Comment by Bluesy1** [comment]
Should we add a footer while already working on the footers then?

**Comment by Bluesy1** [comment]
also, codecov isn't running on this PR, which is normally a blocking issue

**Comment by CasparN** [comment]
> Should we add a footer while already working on the footers then?

I wouldn't know where to add ``embed.set_footer(text="Start playing by typing /programs tictactoe")``

> also, codecov isn't running on this PR, which is normally a blocking issue

How do I enable it / is _really_ needed?

**Comment by Bluesy1** [comment]
> > Should we add a footer while already working on the footers then?
> 
> I wouldn't know where to add `embed.set_footer(text="Start playing by typing /programs tictactoe")`

I'd have to check, but I think it would be in the `view.py` for that sub module, there should be a central callback iirc where the embed is handled, and in `programs.py` where the initial embed is made. You probably could find all spots by looking for where `.set_image(url="attachment://tictactoe.png")` is.

> > also, codecov isn't running on this PR, which is normally a blocking issue
> 
> How do I enable it / is _really_ needed?

It isn't needed yet because i'm still building test coverage, but i would like to know why it isn't running.
Also, I do want to add the footer text to checks for corresponding tests so if i change the layout of command structure later, there's a higher chance of this being caught by the continuous integration

File: charbot/sudoku/view.py [hunk 1]
```diff
@@ -192,7 +192,8 @@ def block_choose_embed(self) -> discord.Embed:
         """
         embed = discord.Embed(title="Sudoku", description=f"```ansi\n{self.puzzle}```", color=discord.Color.blurple())
         embed.set_author(name=self.author.display_name, icon_url=self.author.display_avatar.url)
-        embed.set_footer(text="Play Sudoku by Typing /programs sudoku")
+        main
```
Comment (Bluesy1) [inline charbot/sudoku/view.py]:
```suggestion
```

**Review by Bluesy1 — CHANGES_REQUESTED** [review]
Need to add footer here: https://github.com/CasparN/CharB0T/blob/main/charbot/programs.py#L120

Need to update footer here:
https://github.com/CasparN/CharB0T/blob/main/charbot/programs.py#L149

File: tests/charbot/test_sudoku.py [hunk 2]
```diff
@@ -237,7 +237,7 @@ async def test_view_block_choose_embed(_unused_puzzle_unsolved):
     view = sudoku.Sudoku(_unused_puzzle_unsolved, mock_member, mock_bot)
     embed = view.block_choose_embed()
     assert embed.title == "Sudoku"
-    assert embed.footer.text == "Play Sudoku by Typing /programs sudoku"
+    assert embed.footer.text == "Play by typing /programs sudoku"
```
Comment (Bluesy1) [inline tests/charbot/test_sudoku.py:240]:
```suggestion
    assert embed.footer.text == "Start playing by typing /programs sudoku"
```

**Review by Bluesy1 — CHANGES_REQUESTED** [review]

**Review by Bluesy1 — APPROVED** [review]

**Pull request merged**